themes/bootstrap3/templates/RecordDriver/SolrOverdrive/hold.phtml

```diff
@@ -1,59 +1,81 @@
-         <?php if ($cover): ?>
-             <img alt="Cover Image" class="recordcover" src="<?=$this->escapeHtmlAttr($cover)?>">
-         <?php endif; ?>
-     </div>
      <div class="media-body">
-    <h2>
-        <div class="odbrand pull-left">
-          <img class="media-left" src="https://developerportaldev.blob.core.windows.net/media/Default/images/newLogos/OverDrive_Logo_28x28_rgb.jpg">
-        </div>
-        <div>
-        <?=$this->transEsc($actionTitleCode); ?>
-        </div>
+    <h2><?=$this->transEsc($actionTitleCode); ?>
+        <span class="odbrand pull-left"
+          <img class="media-left" src="<?=$this->imageLink('OverDrive_Logo_28x28_rgb.jpg')?>">
+        </span>
     </h2>
-
+         <div class="pull-right  img-col">
+             <?php if ($cover): ?>
+                 <img alt="<?=$this->transEscAttr('Cover Image')?>" class="recordcover" src="<?=$this->escapeHtmlAttr($cover)?>">
+             <?php endif; ?>
+         </div>
 
     <h3 class="title"><?=$this->escapeHtml($title)?></h3>
 
      <?php if (!empty($listAuthors)): ?>
        <div class="authors">
-         <?=$this->transEsc('by')?>:
+         <?=$this->transEsc('by')?>
          <?=$this->escapeHtml($listAuthors[0])?>
           <?php if (count($listAuthors) > 1): ?>, <?=$this->transEsc('more_authors_abbrev')?><?php endif; ?>
        </div>
      <?php endif; ?>
- </div>
 
     <div class="action <?=$action ?>">
 
     <?php if ($action == 'checkoutConfirm'): ?>
         <?php if ($result->status): ?>
             <div class="odformats">
-                <h3><?=$this->transEsc('od_dl_formats'); ?></h3>
+                <h4><?=$this->transEsc('od_dl_formats'); ?></h4>
                 <?php foreach ($formats as $key => $format): ?>
                 <ul><li><?=$this->translate($format->name) ?></li></ul>
                 <?php endforeach;?>
             </div>
+
             <form class="form-record-hold" method="POST" name="placeHold" data-lightbox-onclose="VuFind.refreshPage">
-            <div class="pull-right">
 
-                <input name="doAction" value="doCheckout" type="hidden">
-                <input class="btn btn-primary" name="doCheckout" value="<?=$this->transEscAttr('od_but_checkout_s'); ?>" type="submit">
+            <?php  if ($result->data->isMagazine): ?>
+              <div class="form-group">
+                <label id="select-issue-label"><?=$this->translate('Issue') ?></label>
+                <select class="form-control" name="edition" id="select-edition" data-open="false">
+
+                <?php foreach ($result->data->products as $key => $issue):
+                    $checkedout = $issue->checkedout ? ' - ' . $this->transEsc('od_mag_issue_ischeckedout') : ''; ?>
+                    <option value="<?=$issue->id?>"><?=$issue->edition . $checkedout?> </option>
+                <?php endforeach;?>
+                </select>
             </div>
+            <?php  endif;?>
```

Comment:
Minor style thing -- might be worth a string replace from `<?php  ` (2 spaces) to `<?php ` (1 space) to clean up some unnecessary whitespace in a couple of spots here. Also worth adding a space before the `?>` in this close tag.

Reply:
I think I caught all of these. Also removed all space after short echo tags based on other templates I looked at.